srcCs/PerftestTransport.cs

```diff
@@ -270,6 +295,23 @@ public string PrintTransportConfigurationSummary()
                 sb.Append("\tNic: ").Append(allowInterfaces).Append("\n");
             }
 
+            sb.Append( "\tUse Multicast: ");
```

Comment:
apply same changes as in cpp

Reply:
Open discussion about if showing always the Multicast information

Reply:
We decided to keep this in all cases.